packages/@aws-cdk/core/lib/construct.ts

```diff
@@ -199,6 +199,10 @@ export class ConstructNode {
 
   /**
    * Returns the child construct that has the id `Default` or `Resource"`
+   * This is usually the construct that provides the bulk of the underlying functionality.
+   * It's typically used as an escape hatch to access the underlying construct for modifications
+   * that are not avialable at the higher levels.
```

Comment:
Typo.

I also wouldn't say "it's typically used as an escape hatch", I would phrase it like:

"You can access it if you need to make modifications that are not available at the higher levels."